corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -228,6 +236,10 @@
         {% endif %}
     </head>
     <body>
+        <!-- Google Tag Manager (noscript) -->
+        <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-52MT7FS"
```

Comment:
Maybe not necessary, given how dysfunctional HQ is if you don't have scripting enabled.